src/components/Register/index.jsx

```diff
@@ -0,0 +1,256 @@
+import { Alert, Button, Checkbox, Input } from "antd";
+import Modal from "antd/lib/modal/Modal";
+import React, { useState } from "react";
+import { validateEmail } from "../../functions";
+import { useTranslation } from "react-i18next";
+
+const Register = (props) => {
+  const [t] = useTranslation();
+  const [registerInformation, setRegisterInformation] = useState({
+    fullname: "",
+    age: "",
+    email: "",
+    password: "",
+    agree: false,
+  });
+  const [errors, setErrors] = useState({});
+  const [termsOpen, setTermsOpen] = useState(false);
+  const handleChange = (key, value) => {
+    let newValues = Object.assign({}, registerInformation);
+    newValues[key] = value;
+    setRegisterInformation(newValues);
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    let { fullname, email, age, password, agree } = registerInformation;
+    let errors = {};
+    if (fullname.trim() === "") {
+      errors["fullname"] = t("register.fullName");
+    }
+    if (email.trim() === "") {
+      errors["email"] = t("register.validEmail");
+    }
+    if (age.trim() === "") {
+      errors["age"] = t("register.age");
+    }
+    if (password.trim().length < 6) {
```

Comment:
magic number

Reply:
😅, shall I make it 8 ? or what?

Reply:
Sorry, I missed this point in the manual. it's updated now.